node/src/Agent/README.TXT

```diff
@@ -0,0 +1,5 @@
+THE TRACE AGENT AND DOGSTATSD MUST BE MANAGED BY THE EXTENSION CODE.
+
+MANUALLY RESTARTING THE DATADOG AGENT PROCESSES IS UNSUPPORTED. 
+
+IN ORDER TO CHANGE CONFIGURATION, THE AZURE APP SERVICE INSTANCE MUST BE FULLY STOPPED AND STARTED.
```

Comment:
Have you confirmed this is still true? I know this used to be the case but I'm not entirely sure that restarts are insufficient.